ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.10.yaml

```diff
@@ -217,8 +217,8 @@ tests:
         limits:
           memory: 6Gi
         requests:
-          cpu: "2"
-          memory: 4Gi
+          cpu: "4"
+          memory: 6Gi
```

Comment:
Have a pointer of the failure that leads to this adjustment (lacking context)?

Reply:
we use same/similar setup on the kafka bits - that's why.

Reply:
Have a pointer for the Kafka bits? (lack context)